assets/js/blocks/product-search/editor.scss

```
@@ -0,0 +1,12 @@
+.wc-block-product-search__field.input-control {
+	-webkit-appearance: textfield;
```

Is this declaration required? Activating/deactivating it didn't make and difference for me.

In case we need it, we could use:
```suggestion
	appearance: textfield;
```
this way Autoprefixer will automatically add the `-moz` and `-webkit` prefixes.